versions/ru/toml-v0.5.0.md

```diff
@@ -0,0 +1,906 @@
+![TOML Logo](../../logos/toml-200.png)
+
+TOML v0.5.0
+===========
+
+Очевидный Минималистичный Язык Тома (Tom's Obvious, Minimal Language).
+
+Автор: Том Престон-Вернер (Tom Preston-Werner).
+
+Версия TOML 0.5.0 должна рассматриваться как предельно стабильная. Цель версии
+1.0.0  быть обратно совместимой с версией 0.5.0 (на сколько это возможно,
+с сохранением читаемости для человека). Авторам TOML имплементаций строго
+рекомендуется реализовать совместимость с версией 0.5.0, что позволит
+без особых затрат выполнить обновление к версии 1.0.0, когда это случиться.
+
+Переводчик
```

Comment:
I think it's irrelevant who the translator is and who he works for. Can we please drop this from translation?